index.html

```diff
@@ -197,9 +209,38 @@ <h2>Professional Art Printing Data</h2>
         >
           See Project
         </button>
-      </article>
-      <article class="professional-art">
-        <h2>Professional Art Printing Data</h2>
+      </div>
+      </div>
+      <div class="container-section2">
```

Comment:
[OPTIONAL] In the desktop view, some of the elements are not correctly centered across the app. We could try to fix these by taking advantage of the flex properties like `justify-content` and `align-items`. 

My guess is that you're using a small screen to develop your app, and while this is not bad at all, designing for every screen is important. My advice in this case is to take advantage of the devtools of the web browser you're using to not only make the screen smaller, but also bigger and emulate bigger screens. [You can take a look at this ranking of common screen sizes to know what to develop for.](https://www.google.com/search?q=rank+of+screen+sizes+desktop&oq=rank+of+screen+sizes+desktop&aqs=chrome..69i57j33i22i29i30.5847j0j1&sourceid=chrome&ie=UTF-8)


![image](https://user-images.githubusercontent.com/79114233/140407974-696a0918-20ed-4a96-b29e-8d0f671253e7.png)

Reply:
Also fixed. Now it looks even better.